### ISSUE TITLE
tilt: missing embedded assets

### ISSUE THREAD

**Issue by jgillich** [NixOS/nixpkgs#260411]
### Describe the bug
Cloud assets were deprecated in 0.33.6, see https://github.com/tilt-dev/tilt/pull/6214

Launching tilt now returns an error:

```
$ tilt up
Tilt started on http://127.0.0.1:10350/ (listening on 0.0.0.0)
v0.33.6, built 
Error: running in prod mode, but JS/CSS files are not available.
Please report this: https://github.com/tilt-dev/tilt/issues
```

I believe the assets need to be built with `make build-js` so they can be embedded.

### Notify maintainers

@anton-dessiatov 

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.5-200.fc38.x86_64, Debian GNU/Linux, 12 (bookworm), nobuild`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.17.0`
 - nixpkgs: `/home/vscode/.nix-defexpr/channels/nixpkgs`
```

**Comment by nicks** [comment]
Sorry for the breakage.

Ya, it needs to build the js assets. There's a good example in the homebrew formula:
https://github.com/Homebrew/homebrew-core/blob/faea31354e37ab5d21a9ce4ebea6bee209f3d9c3/Formula/t/tilt.rb

This will make everything more hermetic and faster overall so it's a good change anyway.

**Comment by anton-dessiatov** [comment]
@jgillich, thanks for reporting this! I haven't checked the package integrity for quite a while so all updates were mostly auto-bumps.

I believe I have fixed the issue at #260712. Could you please take a look and see if the problem is gone for you?
